### PR TITLE
[T-650] Fix number padding in codeframe being incorrect if endLine == 10

### PR DESCRIPTION
**Pull request by DeMoorJasper**
# ↪️ Pull Request

This PR aims to fix a bug with codeframe regarding padding being inconsistent.

The bug was related to the last line in the codeframe being 10, which would result in the last index being 9 which in turn results in the line length being 1 as the index is 9 and not 10... This PR does a `+1` before calculating the lastline number length to ensure it's always the actual number of the line instead of an index.

This was a pretty straightforward thing that should've been in the codeframe test-suite all along, but I overlooked it originally.

### CODE REVIEW

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 6.39s | -1.11s 🚀 |
| Cached | 2.28s | +6.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 261.00ms | +82.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 261.00ms | +82.00ms ⚠️ |
| dist/legacy/index.42516b4a.js | 1.10kb | +0.00b | 1.69s | -109.00ms 🚀 |
| dist/modern/index.8a1ced31.js | 1.10kb | +0.00b | 1.69s | +969.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 680.00ms | -156.00ms 🚀 |
| dist/modern/index.html | 701.00b | +0.00b | 678.00ms | -155.00ms 🚀 |
| dist/legacy/index.0e158b6b.css | 77.00b | +0.00b | 1.69s | -110.00ms 🚀 |
| dist/modern/index.e480862d.css | 77.00b | +0.00b | 1.42s | -385.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 192.00ms | -110.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 191.00ms | -110.00ms 🚀 |
| dist/legacy/index.42516b4a.js | 1.10kb | +0.00b | 461.00ms | -111.00ms 🚀 |
| dist/modern/index.8a1ced31.js | 1.10kb | +0.00b | 460.00ms | -111.00ms 🚀 |
| dist/legacy/index.html | 701.00b | +0.00b | 466.00ms | -108.00ms 🚀 |
| dist/modern/index.html | 701.00b | +0.00b | 460.00ms | -111.00ms 🚀 |
| dist/legacy/index.0e158b6b.css | 77.00b | +0.00b | 461.00ms | -111.00ms 🚀 |
| dist/modern/index.e480862d.css | 77.00b | +0.00b | 460.00ms | -111.00ms 🚀 |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 28.20s | +344.00ms |
| Cached | 2.39s | +1.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 150.00ms | -51.00ms 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.42kb | +0.00b | 356.00ms | +33.00ms ⚠️ |
| dist/PermalinkedComment.c85d137d.js | 4.22kb | +0.00b | 346.00ms | +25.00ms ⚠️ |
| dist/UserProfile.30e15baf.js | 1.62kb | +0.00b | 346.00ms | +25.00ms ⚠️ |
| dist/NotFound.5aa6b041.js | 387.00b | +0.00b | 346.00ms | +25.00ms ⚠️ |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 3.10m | -885.00ms |
| Cached | 5.14s | -140.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/popup.4a66e15e.js | 170.62kb | +0.00b | 59.64s | +20.17s ⚠️ |
| dist/Toolbar.33a83ae3.js | 92.34kb | +0.00b | 1.04m | -3.22s 🚀 |
| dist/card.1007a95f.js | 52.86kb | +0.00b | 1.77m | +41.85s ⚠️ |
| dist/dropzone.c8c03b6a.js | 14.36kb | +0.00b | 1.77m | +42.60s ⚠️ |
| dist/ui.cee430a3.js | 13.97kb | +0.00b | 17.74s | +917.00ms ⚠️ |
| dist/smartMediaEditor.7ee0079c.js | 13.95kb | +0.00b | 1.02m | -4.29s 🚀 |
| dist/workerHasher.65b8a1b7.js | 11.78kb | +0.00b | 17.74s | +1.72s ⚠️ |
| dist/EmojiTypeAheadComponent.9af6e5fc.js | 8.67kb | +0.00b | 15.70s | +891.00ms ⚠️ |
| dist/component.11476a92.js | 6.05kb | +0.00b | 9.17s | +1.49s ⚠️ |
| dist/card.4ac5c1b2.js | 5.61kb | +0.00b | 20.37s | +2.19s ⚠️ |
| dist/EmojiPickerComponent.624bf3f4.js | 3.58kb | +0.00b | 15.68s | -1.50m 🚀 |
| dist/index.393afe59.css | 3.46kb | +0.00b | 6.97s | -59.85s 🚀 |
| dist/dropzone.2f1a1e17.js | 3.23kb | +0.00b | 59.77s | +23.05s ⚠️ |
| dist/16.d1b8e6f7.js | 2.38kb | +0.00b | 7.02s | +2.31s ⚠️ |
| dist/EmojiUploadComponent.88a444df.js | 2.18kb | +0.00b | 15.68s | +870.00ms ⚠️ |
| dist/card.cadccd96.js | 2.03kb | +0.00b | 35.50s | +15.68s ⚠️ |
| dist/16.86a8c7d4.js | 1.66kb | +0.00b | 9.17s | +471.00ms ⚠️ |
| dist/browser.16c438d4.js | 1.61kb | +0.00b | 1.77m | +44.17s ⚠️ |
| dist/workerHasher.9bb425b1.js | 1.54kb | +0.00b | 16.78s | +1.97s ⚠️ |
| dist/workerHasher.f66f00c7.js | 1.54kb | +0.00b | 1.02m | -45.01s 🚀 |
| dist/16.c1ed07f8.js | 1.33kb | +0.00b | 9.17s | +471.00ms ⚠️ |
| dist/16.249c644a.js | 1.31kb | +0.00b | 8.70s | -51.84s 🚀 |
| dist/heading6.4c4017db.js | 1.30kb | +0.00b | 59.22s | +49.84s ⚠️ |
| dist/heading3.93606b9c.js | 1.29kb | +0.00b | 59.22s | +49.84s ⚠️ |
| dist/16.06d5a67b.js | 1.18kb | +0.00b | 8.71s | -51.84s 🚀 |
| dist/heading5.dde15cec.js | 1.17kb | +0.00b | 59.22s | +49.84s ⚠️ |
| dist/16.ecf8a702.js | 1.13kb | +0.00b | 8.31s | -1.23s 🚀 |
| dist/16.da43aa07.js | 1.13kb | +0.00b | 9.21s | +510.00ms ⚠️ |
| dist/16.dd07505e.js | 1.10kb | +0.00b | 4.77s | -1.96s 🚀 |
| dist/heading4.5316f6f2.js | 1.06kb | +0.00b | 59.22s | +49.84s ⚠️ |
| dist/action.cb287eb0.js | 1.00kb | +0.00b | 8.71s | -51.84s 🚀 |
| dist/media-picker-analytics-error-boundary.553055f6.js | 837.00b | +0.00b | 1.00m | +20.76s ⚠️ |
| dist/media-card-analytics-error-boundary.964206cc.js | 833.00b | +0.00b | 19.46s | +2.64s ⚠️ |
| dist/media-viewer-analytics-error-boundary.bbe40a02.js | 757.00b | +0.00b | 35.40s | +15.74s ⚠️ |
| dist/EmojiPickerComponent.a968dafe.js | 535.00b | +0.00b | 15.68s | +870.00ms ⚠️ |
| dist/simpleHasher.f8e69464.js | 528.00b | +0.00b | 17.74s | +917.00ms ⚠️ |
| dist/simpleHasher.32d9352e.js | 528.00b | +0.00b | 1.02m | -45.01s 🚀 |
| dist/index.html | 119.00b | +0.00b | 2.94s | -2.74s 🚀 |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.d6c6b729.js | 2.73mb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/pdfRenderer.cd742f5e.js | 1.11mb | -4.00b 🚀 | 659.00ms | +32.00ms |
| dist/Modal.e3bd77d0.js | 41.40kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/component.59edff1f.js | 30.78kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/component.37867f50.js | 22.51kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/js.48be70a4.js | 16.40kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/workerHasher.65b8a1b7.js | 11.78kb | +0.00b | 660.00ms | +33.00ms |
| dist/component.11476a92.js | 6.05kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/png-chunks-extract.8d15a914.js | 3.41kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/Modal.2835744c.js | 3.08kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.d1b8e6f7.js | 2.38kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/ResourcedEmojiComponent.d9a49606.js | 2.00kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/16.f40c1329.js | 1.73kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.86a8c7d4.js | 1.66kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.cff410ff.js | 1.38kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.c1ed07f8.js | 1.33kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.6603f2f3.js | 1.33kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.a4b392c2.js | 1.32kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/16.249c644a.js | 1.31kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.ab5c0088.js | 1.27kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.afee1494.js | 1.23kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.e60b4d77.js | 1.21kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/16.7c7a40c9.js | 1.19kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.06d5a67b.js | 1.18kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.2096b9f3.js | 1.14kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/16.f040d62d.js | 1.14kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.ecf8a702.js | 1.13kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.1b342686.js | 1.13kb | +0.00b | 667.00ms | +39.00ms ⚠️ |
| dist/16.2e7f52c6.js | 1.13kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/16.da43aa07.js | 1.13kb | +0.00b | 667.00ms | +38.00ms ⚠️ |
| dist/16.dd07505e.js | 1.10kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/component.40194190.js | 1.07kb | +0.00b | 668.00ms | +39.00ms ⚠️ |
| dist/16.317b9164.js | 1.05kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/action.cb287eb0.js | 1.00kb | +0.00b | 666.00ms | +37.00ms ⚠️ |
| dist/index.html | 119.00b | +0.00b | 671.00ms | +40.00ms ⚠️ |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/270755357158015500)**

**Comment by parcel-benchmark** [comment]
## Benchmark Results
<details><summary>Kitchen Sink ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 5.99s | -284.00ms |
| Cached | 2.16s | +12.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 210.00ms | -19.00ms 🚀 |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 210.00ms | -18.00ms 🚀 |
| dist/legacy/index.42516b4a.js | 1.10kb | +0.00b | 1.50s | -109.00ms 🚀 |
| dist/modern/index.8a1ced31.js | 1.10kb | +0.00b | 1.50s | -108.00ms 🚀 |
| dist/legacy/index.html | 701.00b | +0.00b | 1.50s | +862.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 759.00ms | +124.00ms ⚠️ |
| dist/legacy/index.0e158b6b.css | 77.00b | +0.00b | 1.28s | -334.00ms 🚀 |
| dist/modern/index.e480862d.css | 77.00b | +0.00b | 1.27s | +112.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/legacy/parcel.d5807e82.webp | 102.94kb | +0.00b | 271.00ms | +131.00ms ⚠️ |
| dist/modern/parcel.d5807e82.webp | 102.94kb | +0.00b | 271.00ms | +131.00ms ⚠️ |
| dist/legacy/index.42516b4a.js | 1.10kb | +0.00b | 492.00ms | +175.00ms ⚠️ |
| dist/modern/index.8a1ced31.js | 1.10kb | +0.00b | 492.00ms | +174.00ms ⚠️ |
| dist/legacy/index.html | 701.00b | +0.00b | 494.00ms | +175.00ms ⚠️ |
| dist/modern/index.html | 701.00b | +0.00b | 492.00ms | +175.00ms ⚠️ |
| dist/legacy/index.0e158b6b.css | 77.00b | +0.00b | 493.00ms | +175.00ms ⚠️ |
| dist/modern/index.e480862d.css | 77.00b | +0.00b | 492.00ms | +175.00ms ⚠️ |

</p></details>

<details><summary>React HackerNews ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 25.30s | -658.00ms |
| Cached | 2.24s | -115.00ms 🚀 |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 204.00ms | +11.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.js | 478.42kb | +0.00b | 272.00ms | -74.00ms 🚀 |
| dist/PermalinkedComment.c85d137d.js | 4.22kb | +0.00b | 270.00ms | -73.00ms 🚀 |
| dist/UserProfile.30e15baf.js | 1.62kb | +0.00b | 270.00ms | -73.00ms 🚀 |
| dist/NotFound.5aa6b041.js | 387.00b | +0.00b | 270.00ms | -73.00ms 🚀 |
| dist/logo.24c8bf9e.png | 274.00b | +0.00b | 225.00ms | -43.00ms 🚀 |

</p></details>

<details><summary>AtlasKit Editor ✅</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | 2.76m | -1.50s |
| Cached | 5.00s | -86.00ms |

#### Cold Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/pdfRenderer.cd742f5e.js | 1.11mb | +0.00b | 51.33s | -2.72s 🚀 |
| dist/popup.4a66e15e.js | 170.62kb | +0.00b | 51.10s | +18.70s ⚠️ |
| dist/media-viewer.1d9322e7.js | 69.09kb | +0.00b | 15.34s | -1.07s 🚀 |
| dist/card.ce67dc35.js | 55.09kb | +0.00b | 12.74s | -882.00ms 🚀 |
| dist/card.1007a95f.js | 52.86kb | +0.00b | 52.77s | -3.82s 🚀 |
| dist/Modal.e3bd77d0.js | 41.40kb | +0.00b | 5.26s | -753.00ms 🚀 |
| dist/component.37867f50.js | 22.51kb | +0.00b | 5.25s | -765.00ms 🚀 |
| dist/js.48be70a4.js | 16.40kb | +0.00b | 5.25s | -766.00ms 🚀 |
| dist/dropzone.c8c03b6a.js | 14.36kb | +0.00b | 52.22s | -4.36s 🚀 |
| dist/ui.cee430a3.js | 13.97kb | +0.00b | 12.61s | -980.00ms 🚀 |
| dist/smartMediaEditor.7ee0079c.js | 13.95kb | +0.00b | 54.65s | -41.98s 🚀 |
| dist/workerHasher.65b8a1b7.js | 11.78kb | +0.00b | 54.21s | +41.39s ⚠️ |
| dist/EmojiTypeAheadComponent.9af6e5fc.js | 8.67kb | +0.00b | 54.21s | +42.44s ⚠️ |
| dist/component.11476a92.js | 6.05kb | +0.00b | 5.26s | -755.00ms 🚀 |
| dist/card.4ac5c1b2.js | 5.61kb | +0.00b | 14.44s | -1.17s 🚀 |
| dist/media-viewer.bc661bce.js | 3.92kb | +0.00b | 15.33s | -1.07s 🚀 |
| dist/EmojiPickerComponent.624bf3f4.js | 3.58kb | +0.00b | 9.80s | -1.45s 🚀 |
| dist/index.393afe59.css | 3.46kb | +0.00b | 1.59m | +36.40s ⚠️ |
| dist/png-chunks-extract.8d15a914.js | 3.41kb | +0.00b | 5.25s | -765.00ms 🚀 |
| dist/Modal.2835744c.js | 3.08kb | +0.00b | 4.22s | -622.00ms 🚀 |
| dist/EmojiUploadComponent.88a444df.js | 2.18kb | +0.00b | 54.21s | +42.44s ⚠️ |
| dist/card.cadccd96.js | 2.03kb | +0.00b | 29.19s | +10.30s ⚠️ |
| dist/ResourcedEmojiComponent.d9a49606.js | 2.00kb | +0.00b | 4.22s | -583.00ms 🚀 |
| dist/date.d49e64f8.js | 1.73kb | +0.00b | 5.95s | -650.00ms 🚀 |
| dist/16.f40c1329.js | 1.73kb | +0.00b | 5.76s | -666.00ms 🚀 |
| dist/images.80270b1f.js | 1.67kb | +0.00b | 6.12s | -583.00ms 🚀 |
| dist/feedback.8b7c7928.js | 1.64kb | +0.00b | 6.60s | -452.00ms 🚀 |
| dist/workerHasher.9bb425b1.js | 1.54kb | +0.00b | 54.21s | +42.44s ⚠️ |
| dist/workerHasher.f66f00c7.js | 1.54kb | +0.00b | 54.51s | -42.12s 🚀 |
| dist/list-number.4632b459.js | 1.46kb | +0.00b | 6.28s | -528.00ms 🚀 |
| dist/status.cccf04be.js | 1.46kb | +0.00b | 6.50s | -431.00ms 🚀 |
| dist/16.cff410ff.js | 1.38kb | +0.00b | 5.43s | -793.00ms 🚀 |
| dist/code.af9ca2b3.js | 1.38kb | +0.00b | 5.95s | -647.00ms 🚀 |
| dist/16.6603f2f3.js | 1.33kb | +0.00b | 5.43s | -793.00ms 🚀 |
| dist/16.249c644a.js | 1.31kb | +0.00b | 5.93s | -662.00ms 🚀 |
| dist/link.d12cc130.js | 1.30kb | +0.00b | 6.28s | -528.00ms 🚀 |
| dist/heading6.4c4017db.js | 1.30kb | +0.00b | 6.60s | -452.00ms 🚀 |
| dist/heading3.93606b9c.js | 1.29kb | +0.00b | 6.50s | -570.00ms 🚀 |
| dist/16.ab5c0088.js | 1.27kb | +0.00b | 5.78s | -653.00ms 🚀 |
| dist/emoji.e603e70e.js | 1.23kb | +0.00b | 6.11s | -587.00ms 🚀 |
| dist/16.afee1494.js | 1.23kb | +0.00b | 5.76s | -666.00ms 🚀 |
| dist/16.7c7a40c9.js | 1.19kb | +0.00b | 5.76s | -666.00ms 🚀 |
| dist/16.06d5a67b.js | 1.18kb | +0.00b | 5.93s | -662.00ms 🚀 |
| dist/heading5.dde15cec.js | 1.17kb | +0.00b | 6.60s | -452.00ms 🚀 |
| dist/expand.6889aead.js | 1.16kb | +0.00b | 6.60s | -3.35s 🚀 |
| dist/16.f040d62d.js | 1.14kb | +0.00b | 5.43s | -789.00ms 🚀 |
| dist/16.1b342686.js | 1.13kb | +0.00b | 5.43s | -793.00ms 🚀 |
| dist/16.2e7f52c6.js | 1.13kb | +0.00b | 5.43s | -789.00ms 🚀 |
| dist/heading2.e219309a.js | 1.11kb | +0.00b | 6.50s | -432.00ms 🚀 |
| dist/16.dd07505e.js | 1.10kb | +0.00b | 4.22s | -584.00ms 🚀 |
| dist/component.40194190.js | 1.07kb | +0.00b | 4.22s | -583.00ms 🚀 |
| dist/mention.b1828447.js | 1.07kb | +0.00b | 6.28s | -530.00ms 🚀 |
| dist/heading4.5316f6f2.js | 1.06kb | +0.00b | 6.60s | -452.00ms 🚀 |
| dist/16.317b9164.js | 1.05kb | +0.00b | 5.78s | -652.00ms 🚀 |
| dist/layout.ac6734f9.js | 1.05kb | +0.00b | 6.12s | -586.00ms 🚀 |
| dist/divider.d2be55a1.js | 1.03kb | +0.00b | 6.11s | -587.00ms 🚀 |
| dist/quote.f4e09143.js | 1.02kb | +0.00b | 6.43s | -502.00ms 🚀 |
| dist/action.cb287eb0.js | 1.00kb | +0.00b | 5.93s | -662.00ms 🚀 |
| dist/decision.91f3af14.js | 1010.00b | +0.00b | 6.11s | -587.00ms 🚀 |
| dist/panel-warning.68b04096.js | 1008.00b | +0.00b | 6.43s | -47.61s 🚀 |
| dist/list.057cb664.js | 978.00b | +0.00b | 6.28s | -529.00ms 🚀 |
| dist/heading1.fa66b375.js | 976.00b | +0.00b | 6.50s | -432.00ms 🚀 |
| dist/panel-error.c4b05f6f.js | 906.00b | +0.00b | 6.28s | -532.00ms 🚀 |
| dist/panel.9dbc048e.js | 897.00b | +0.00b | 6.43s | -47.61s 🚀 |
| dist/table.b35545be.js | 889.00b | +0.00b | 6.50s | -432.00ms 🚀 |
| dist/panel-success.7eb0809a.js | 845.00b | +0.00b | 6.43s | -47.61s 🚀 |
| dist/panel-note.ec8386b7.js | 841.00b | +0.00b | 6.43s | -47.61s 🚀 |
| dist/media-card-analytics-error-boundary.964206cc.js | 833.00b | +0.00b | 12.61s | -980.00ms 🚀 |
| dist/media-card-analytics-error-boundary.57503222.js | 833.00b | +0.00b | 52.90s | -43.74s 🚀 |
| dist/media-viewer-analytics-error-boundary.bbe40a02.js | 757.00b | +0.00b | 15.34s | -1.07s 🚀 |
| dist/EmojiPickerComponent.a968dafe.js | 535.00b | +0.00b | 9.80s | -1.97s 🚀 |
| dist/simpleHasher.f8e69464.js | 528.00b | +0.00b | 12.61s | -979.00ms 🚀 |
| dist/simpleHasher.32d9352e.js | 528.00b | +0.00b | 54.52s | -42.12s 🚀 |
| dist/index.html | 119.00b | +0.00b | 853.00ms | +73.00ms ⚠️ |

#### Cached Bundles

| Bundle | Size | Difference | Time | Difference |
| --- | --- | --- | --- | --- |
| dist/index.d6c6b729.js | 2.73mb | -3.00b 🚀 | 651.00ms | -18.00ms |

</p></details>

<details><summary>Three.js x4 🚨</summary><p>

#### Timings

| Description | Time | Difference |
| --- | --- | --- |
| Cold | **FAILED** | -0.00ms |
| Cached | **FAILED** | -0.00ms |

#### Cold Bundles

*No bundles found, this is probably a failed build...*
#### Cached Bundles

*No bundles found, this is probably a failed build...*
</p></details>

**[Click here to view a detailed benchmark overview.](https://benchmark.parceljs.org/benchmark/270755374175355400)**

File: packages/core/codeframe/test/codeframe.test.js [hunk 1]
```diff
@@ -752,4 +754,46 @@ describe('codeframe', () => {
...
+    assert.equal(
+      lines[1],
+      `> 8  | import { unstable_CompositeItem as CompositeItem } from 'reakit/Composite';`,
+    );
+    assert.equal(
+      lines[2],
+      '>    |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^',
+    );
+    assert.equal(lines[3], '  9  | /**');
+    assert.equal(lines[4], '  10 |  * Internal dependencies');
```
Comment (mischnic) [inline packages/core/codeframe/test/codeframe.test.js:797]: Line numbers are usually right-aligned:
```
 9 | /**
10 |  * Internal dependencies
```

Comment (DeMoorJasper) [reply packages/core/codeframe/test/codeframe.test.js:797]: you're right... it should be a padStart

Will update it, shouldn't be that much work

File: packages/core/codeframe/test/fixtures/wordpress-components.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import _extends from "@babel/runtime/helpers/esm/extends";
+import _objectWithoutProperties from "@babel/runtime/helpers/esm/objectWithoutProperties";
+import { createElement } from "@wordpress/element";
+
+/**
+ * External dependencies
+ */
+import { unstable_CompositeItem as CompositeItem } from 'reakit/Composite';
```
Comment (mischnic) [inline packages/core/codeframe/test/fixtures/wordpress-components.js:8]: Can you remove everything below line 10? Technically this is license infringement, but just with the imports, it should be fine.

Comment (DeMoorJasper) [reply packages/core/codeframe/test/fixtures/wordpress-components.js]: Just replaced it with dummy code, the bug had nothing to do with the actual code anyway

**Pull request merged**